docs/_docs/memory-configuration/data-regions.adoc

```diff
@@ -82,3 +82,72 @@ include::code-snippets/dotnet/MemoryArchitecture.cs[tag=mem,indent=0]
 tab:C++[unsupported]
 --
 
+== Cache Warm-Up Strategy
+
+The cache warm-up strategy provides an ability to load data from the disk to the node before it is joined to the cluster.
+This prevents performance loss when the node is restarted.
+
+[NOTE]
+====
+The cache warm-up strategy is applicable only to the data regions. It can be configured both for all regions (by default) or for each region separately.
+====
+
+Presently, Ignite warm-up strategy implies loading data into the region, starting with indexes, until it runs out of free space.
+
+To warm up all data regions, pass the configuration parameter `LoadAllWarmUpStrategy` to the `DataStorageConfiguration#setDefaultWarmUpConfiguration` as follows:
+
+[source, java]
+----
+setDefaultWarmUpConfiguration(loadAllWarmUpStrategy)
+----
+
+To warm up a specific data region, pass the configuration parameter `LoadAllWarmUpStrategy` to the `DataStorageConfiguration#setWarmUpConfiguration` as follows:
+
+[source, java]
+----
+setWarmUpConfiguration(loadAllWarmUpStrategy)
+----
+
+To stop warming up all data regions, pass the configuration parameter `NoOpWarmUpStrategy` to the `DataStorageConfiguration#setDefaultWarmUpConfiguration` as follows:
```

Comment:
Hello everyone! Yes **NoOpWarmUpStrategy** strategy allows you to disable warm-up regions. But it is recommended to use it only for **setWarmUpConfiguration** and for **setDefaultWarmUpConfiguration** it does not make sense since this is the default behavior (do not warm up anything). 
Example: we have 10 persistent regions, and we set _setDefaultWarmUpConfiguration ->LoadAllWarmUpStrategy_, then to disable the warm-up of specific regions, we must set them _setWarmUpConfiguration -> NoOpWarmUpStrategy_.